superset/assets/src/components/TableSelector.jsx

```diff
@@ -203,6 +205,33 @@ export default class TableSelector extends React.PureComponent {
         {db.database_name}
       </span>);
   }
+  renderTableOption({ focusOption, focusedOption, key, option, selectValue, style, valueArray }) {
+    const classNames = ['Select-option'];
+    if (option === focusedOption) {
+      classNames.push('is-focused');
+    }
+    if (valueArray.indexOf(option) >= 0) {
+      classNames.push('is-selected')
+    }
+    return (
+      <div
+        className={classNames.join(' ')}
+        key={key}
+        onClick={() => selectValue(option)}
```

Comment:
Let's pull this out as a const so that we don't create a new object each time.

Reply:
The problem is that here `selectValue` (and `focusOption` below) are passed as arguments in the function, so we need to create the object every time. :-/